gammapy/makers/background/reflected.py

```diff
@@ -467,21 +467,37 @@ def make_counts_off(self, dataset, observation):
         energy_axis = geom.axes["energy"]
         events = observation.events
 
-        is_point_like = observation.rad_max is not None
-
-        if is_point_like and not geom.is_all_point_sky_regions:
-            raise ValueError(
-                "Must use PointSkyRegion on region in point-like analysis,"
-                f" got {type(geom.region)} instead"
-            )
+        is_point_like = False
+
+        if observation.rad_max is not None:
+            if isinstance(self.region_finder, ReflectedRegionsFinder):
```

Comment:
I think the additional check for the `ReflectedRegionsFinder` is not needed, it should work fine with the `WhobbleRegionsFinder` as well, as long as the circle is consistent with the single rad max value.